crates/y-sweet-worker/src/config.rs

```diff
@@ -10,6 +10,7 @@ const CHECKPOINT_FREQ_SECONDS: &str = "CHECKPOINT_FREQ_SECONDS";
 const S3_ACCESS_KEY_ID: &str = "AWS_ACCESS_KEY_ID";
 const S3_SECRET_ACCESS_KEY: &str = "AWS_SECRET_ACCESS_KEY";
 const S3_REGION: &str = "AWS_REGION";
+const S3_ENDPOINT: &str = "S3_ENDPOINT";
```

Comment:
AWS uses `AWS_ENDPOINT_URL`

https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html

Reply:
I was very confused parsing that line in the docs lol, yes that makes sense.